c7n/resources/elasticbeanstalk.py

```diff
@@ -32,3 +35,48 @@ class resource_type(object):
         )
         filter_name = 'ApplicationNames'
         filter_type = 'list'
+
+@resources.register('elasticbeanstalk-environment')
+class ElasticBeanstalkEnvironment(QueryResourceManager):
+    """ Resource manager for Elasticbeanstalk Environments
+    """
+
+    class resource_type(object):
+        service = 'elasticbeanstalk'
+        enum_spec = ('describe_environments', 'Environments', None)
+        name = id = "EnvironmentName"
+        dimension = None
+        default_report_fields = (
+            'EnvironmentName',
+            'DateCreated',
+            'DateUpdated',
+            )
+        filter_name = 'EnvironmentNames'
+        filter_type = 'list'
+
+    filter_registry = eb_env_filters
+
+@eb_env_filters.register('environment-uptime')
+class EnvironmentUptimeFilter(AgeFilter):
```

Comment:
we actually folded the date comparison functionality directly into the value filter with value_type: age  and just point `key` at any date attribute. so we really don't need a separate filter for it to write a policy like the example here.

Reply:
Oh sweet! I'll remove that then. Thanks for the quick response.